src/olympia/editors/helpers.py

```diff
@@ -358,22 +358,20 @@ def get_actions(self, request, addon):
             # ReviewHelper for its `handler` attribute and we don't care about
             # the actions.
             return actions
-        latest_version = addon.find_latest_version(
-            channel=amo.RELEASE_CHANNEL_LISTED)
         reviewable_because_complete = addon.status not in (
             amo.STATUS_NULL, amo.STATUS_DELETED)
         reviewable_because_admin = (
             not addon.admin_review or
             acl.action_allowed(request, 'ReviewerAdminTools', 'View'))
         reviewable_because_submission_time = (
             not is_limited_reviewer(request) or
-            (latest_version is not None and
-                latest_version.nomination is not None and
-                (datetime.datetime.now() - latest_version.nomination >=
+            (self.version is not None and
+                self.version.nomination is not None and
+                (datetime.datetime.now() - self.version.nomination >=
                     datetime.timedelta(hours=REVIEW_LIMITED_DELAY_HOURS))))
         reviewable_because_pending = (
-            latest_version is not None and
-            len(latest_version.is_unreviewed) > 0)
+            self.version is not None and
+            len(self.version.is_unreviewed) > 0)
```

Comment:
I don't feel like it's an issue - if you can get a file in that state it will break a lot more stuff (since we rely on the fact that purely unlisted add-ons always have `status=amo.STATUS_NULL`)